src/main/java/io/airlift/units/DataSize.java

```diff
@@ -209,7 +209,7 @@ public DataSize convertToMostSuccinctDataSize()
     @JsonValue
     public String toBytesValueString()
     {
-        return Long.toString(bytes) + Unit.BYTE.getUnitString();
+        return bytes + Unit.BYTE.getUnitString();
```

Comment:
Looks unrelated to "Avoid extra string allocation in DataSize parsing". Separate commit?

Reply:
Updated commit message to mention this optimization as well instead.